src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/NPathComplexityCheck.java

```diff
@@ -119,14 +119,51 @@
  * &lt;module name="NPathComplexity"/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public void foo() { // NPath Complexity = 3 (OK)
+ *   if (a &gt; 10) {
+ *     if (a &gt; b) { // +1
```

Comment:
Please remove `// + 1` comments. It is not clear what do they mean.

Reply:
done